pkg/kubelet/BUILD

```diff
@@ -12,12 +12,12 @@ go_library(
         "active_deadline.go",
```

Comment:
Yeah, and this time I moved the code into its own package so it's not easy to review and track the comments...